### PR TITLE
[Notifier] Add docs for Microsoft Teams Options

### PR DESCRIPTION
**Pull request by OskarStark**
Docs for symfony/symfony#40738

### CODE REVIEW

File: notifier/chatters.rst [hunk 1]
```diff
@@ -180,6 +180,44 @@ to add `message options`_::
...
     $chatter->send($chatMessage);
 
+Adding Options to a Microsoft Teams Message
+-------------------------------------------
+
+With a Microsoft Teams message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\MicrosoftTeams\\MicrosoftTeamsOptions` class
+to add `options`_::
+
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Reply\Markup\Button\InlineKeyboardButton;
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Section;
```

File: notifier/chatters.rst [hunk 2]
```diff
@@ -180,6 +180,44 @@ to add `message options`_::
...
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Reply\Markup\InlineKeyboardMarkup;
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\ImageSectionElement;
```

File: notifier/chatters.rst [hunk 3]
```diff
@@ -180,6 +180,44 @@ to add `message options`_::
...
+Adding Options to a Microsoft Teams Message
+-------------------------------------------
+
+With a Microsoft Teams message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\MicrosoftTeams\\MicrosoftTeamsOptions` class
+to add `options`_::
+
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Section;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\ImageSectionElement;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsOptions;
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsOptions;
    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\ImageSectionElement;    
    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Section;
```

File: notifier/chatters.rst [hunk 4]
```diff
@@ -180,6 +180,44 @@ to add `message options`_::
...
+    );
+
+    // Add the custom options to the chat message and send the message
+    $chatMessage->options($options);
+
+    $chatter->send($chatMessage);
+
+.. versionadded:: 5.3
+
+    Options for Microsoft Teams were introduced in Symfony 5.3
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
    Options for Microsoft Teams were introduced in Symfony 5.3.
```

**Comment by javiereguiluz** [comment]
Thanks for this contribution!

However, I find the name "options" confusing. In the docs that you linked, they call them "fields" (and sometimes, "properties"). I find it confusing because things like "title" or "bgColor" doesn't seem options, but data/metadata. An option would be used to decide if the title is displayed or not, but the title content is not an option, right?

Maybe I'm missing things because I don't use Microsoft Teams.

**Comment by OskarStark** [comment]
In notifier we use options term in general 🧐

**Comment by javiereguiluz** [comment]
Oskar, I wasn't aware of that. Then, let's keep it. Better be consistent at least. Thanks!

**Comment by OskarStark** [comment]
> Oskar, I wasn't aware of that. Then, let's keep it. Better be consistent at least. Thanks!

No problem 😃

**Review by javiereguiluz — APPROVED** [review]
Nice contribution! Thanks Oskar.

**Comment by OskarStark** [comment]
Superseeded by https://github.com/symfony/symfony-docs/pull/15288

**Pull request closed without merging**